docs/sphinx/dev_guide/build_configurations.rst

```diff
@@ -0,0 +1,155 @@
+its tests, example codes, etc.  For example,
+
+.. code-block:: bash
+
+  $ ./scripts/lc-builds/toss3_clang10.0.1.sh
+  $ cd build_lc_toss3-clang-10.0.1
+  $ make -j
+  $ make test
+
+Eventually, these scripts may go away and be superceded by the Spack-based
+host-config file generation process when that achieves the level of
+compiler coverage that the scripts have.
+
+
+============================
+Generated Host-Config Files
+============================
+
+The RAJA repository contains two submodules 
+`uberenv <https://github.com/LLNL/uberenv>`_ and
+`radiuss-spack-configs <https://github.com/LLNL/radiuss-spack-configs>`_ that 
+work together to generate host-config files. These are projects in the 
+GitHub LLNL organization and contain utilities shared by various projects. 
+The main uberenv script can be used to drive Spack to generate a *host-config* 
+file that contains all the information required to define a RAJA build 
+environment. The host-config file can then be passed to CMake using the '-C' 
+option to create a build configuration. *Spack specs* defining compiler 
+configurations are maintained in files in the radiuss-spack-configs 
+repository.
+
+RAJA shares its uberenv workflow with other projects. The documentation 
+for this is available in `RADIUSS Uberenv Guide <https://radiuss-ci.readthedocs.io/en/latest/uberenv.html#uberenv-guide>`_.
+
+
+Generating a RAJA host-config file
+------------------------------------
+
+This section describes the host-config file generation process for RAJA.
+
+Machine specific configurations
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Compiler configurations for Livermore computer platforms are contained in 
+in sub-directories in the RAJA ``scripts/uberenv/spack_configs`` directory:
+
+.. code-block:: bash
+
+  $ ls -c1 ./scripts/uberenv/spack_configs
+  blueos_3_ppc64le_ib
+  darwin
+  toss_3_x86_64_ib
+  blueos_3_ppc64le_ib_p9
+  config.yaml
+
+To see currently supported configurations, please see the contents of the 
+``compilers.yaml`` file in each of these sub-directories.
+
+Generating a host-config file
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The main uberenv python script can be invoked from the top-level RAJA directory
+to generate a host-config file for a desired configuration. For example,
+
+.. code-block:: bash
+
+  $ python ./scripts/uberenv/uberenv.py --spec="%gcc@8.1.0"
+  $ python ./scripts/uberenv/uberenv.py --spec="%gcc@8.1.0~shared+openmp tests=benchmarks"
+
+Each command generates a corresponding host-config file in the top-level RAJA 
+directory. The file name contains the platform and OS to which it applies, and 
+the compiler and version. For example,
+
+.. code-block:: bash
+
+  hc-quartz-toss_3_x86_64_ib-gcc@8.1.0-fjcjwd6ec3uen5rh6msdqujydsj74ubf.cmake
+
+Specs that are exercised during the Gitlab CI process are found YAML files in 
+the ``RAJA/.gitlab`` directory. See :ref:`vettedspecs-label` for more 
```

Comment:
I don't think the ref works..

Reply:
Either that, or it doesn't play nicely with github and when i click it, it doesn't show up right. If you double check it, does it work ok?

Reply:
Ok I'll stop commenting a bunch lol, but the problem may be an extra space on the label?

Reply:
It's a readthedocs link: https://raja.readthedocs.io/en/task-devguide-gitlab/sphinx/dev_guide/build_configurations.html

Reply:
Ah! So it does work. Just github can't handle it. Gottcha lol thanks

Reply:
What is the question here? I believe all the links are correct on RTD. I double-checked them.

Reply:
Yes they are. I was checking it from github, and github doesn't understand RTD links. I thought it might be an incorrect link, but it shows up just fine on RTD. So, false alarm!